packages/account-v2/src/components/CountrySelector/__test__/CountrySelector.spec.tsx

```diff
@@ -14,7 +14,9 @@ const mockUseResidenceList = useResidenceList as jest.MockedFunction<typeof useR
 const renderComponents = () => {
     render(
         <APIProvider>
-            <CountrySelector label='Country' name='country' />
+            <AuthProvider>
```

Comment:
CountrySelector component doesn't have any Authorized API calls. Do we still need to wrap<AuthProvider> here in its spec file?

Reply:
I'm adding it everywhere APIProvider is used as a default option, as the whole app expects user to be authorised (maybe apart from login page in legacy code). All the code running in accounts-v2 (and all -v2 packages) is running in the context of AuthProvider / authorised user. 

There are chances that this test will still work without it AuthProvider - but since the whole app is running in context of it, I didn't wanted to spend time to manually verify all the components/tests I've updated :)